rubin_sim/maf/batches/glance_batch.py

```diff
@@ -144,7 +144,7 @@ def glanceBatch(
         bundle_list.append(bundle)
 
     # alt az of long gaps
-    sql = "note = 'long'"
+    sql = "scheduler_note = 'long'"
```

Comment:
Sigh, I see your point about scheduler_note vs note being embedded in many places. 
And true, a whole separate colmap just for this one change is likely overkill.